modules/auxiliary/scanner/ssh/fortinet_backdoor.rb

```diff
@@ -48,14 +48,15 @@ def run_host(ip)
     factory = ssh_socket_factory
 
     ssh_opts = {
-      port:            rport,
+      :port            => rport,
```

Comment:
Is there a reason for changing the syntax here? I prefer the newer hash syntax if possible. All these keys are symbols, too.

Reply:
That said, hashrockets are more consistent with varying key types. I won't complain too much - just asking why it's necessary to change syntax.

Reply:
Consistency is the only reason. While I also prefer the new syntax, rocket formatted ones are what I’ve been asked for before when building exploit information hashes and was the style of the LoginScanner. As such, while I prefer the newer one, I think consistency is more important than minimal changes. I’m open though to other opinions if we want to use them, but stylistically I think the framework would benefit from a more defined code style guideline around hash structures

Reply:
Agreed. I ask only because you're modifying my module. :-)

Reply:
I think hashrockets will be the way to go if we publish our own style guide (and not just cargo-cult the Ruby style guide). We have such wildly varying hashes that it'd make sense to stick to one style.

Reply:
Python is a little more predictable. :-)